.gitmodules

```diff
@@ -7,3 +7,4 @@
 [submodule "shared/gapic-showcase"]
 	path = shared/gapic-showcase
 	url = git@github.com:googleapis/gapic-showcase.git
+	ignore = dirty
```

Comment:
FYI, adding `dirty` here, since `gapic-showcase` has `googleapis` submodule under it as well. 

The version that has is published to `repo:gapic-showcase` doesnt have `sbom.proto`, which was making some tests fail: https://github.com/googleapis/gapic-generator-ruby/actions/runs/8381034866